frontend/src/components/treecluster/TreeClusterDashboard.tsx

```diff
@@ -30,10 +30,17 @@ const TreeClusterDashboard = ({ clusterId }: TreeClusterDashboardProps) => {
             Bewässerungsgruppe: {treecluster.name}
           </h1>
           <p className="mb-4">{treecluster.description}</p>
-          <GeneralLink
+          {treecluster.trees?.length > 0 ? (
+            <GeneralLink
               url={`/map?lat=${treecluster.latitude}&lng=${treecluster.longitude}&zoom=16&cluster=${treecluster.id}`}
               label="Auf der Karte anzeigen"
             />
+          ) : (
```

Comment:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/aa2707e5-1a62-4afe-901e-cc24fa31f622">

Maybe more like this? Otherwise it looks weird if the cluster also has a description.